lib/sphinx_rtm.ex

```diff
@@ -14,6 +16,7 @@ defmodule SphinxRtm do
   end
 
   def handle_event(message = %{type: "message"}, slack, state) do
+    Logger.info("Processing message from #{message.user}")
```

Comment:
If you want to, use SlackUtils.get_user_name(message.user) would show name instead of id ;)